### PR TITLE
python310Packages.mocket: 3.10.9 -> 3.11.0 

### PR DESCRIPTION
**Pull request by fabaff**
###### Description of changes
Changelog: https://github.com/mindflayer/python-mocket/releases/tag/3.11.0

Fix build (https://hydra.nixos.org/build/209257363)

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 217295` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.pytest-ansible</li>
    <li>python311Packages.pytest-ansible</li>
  </ul>
</details>
<details>
  <summary>25 packages failed to build:</summary>
  <ul>
    <li>ansible (python310Packages.ansible-core)</li>
    <li>ansible-later (python310Packages.ansible-later)</li>
    <li>ansible-lint (python310Packages.ansible-lint)</li>
    <li>ansible_2_12</li>
    <li>ansible_2_13</li>
    <li>kargo</li>
    <li>parsedmarc (python310Packages.parsedmarc)</li>
    <li>python310Packages.ansible</li>
    <li>python310Packages.ansible-compat</li>
    <li>python310Packages.ansible-kernel</li>
    <li>python310Packages.ansible-runner</li>
    <li>python310Packages.geoip2</li>
    <li>python310Packages.sopel</li>
    <li>ttp (python310Packages.ttp)</li>
    <li>python311Packages.ansible</li>
    <li>python311Packages.ansible-compat</li>
    <li>python311Packages.ansible-core</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-later</li>
    <li>python311Packages.ansible-lint</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.geoip2</li>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.sopel</li>
    <li>python311Packages.ttp</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.mocket</li>
    <li>python311Packages.mocket</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 217295` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.pytest-ansible</li>
    <li>python311Packages.pytest-ansible</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python311Packages.sopel</li>
  </ul>
</details>
<details>
  <summary>26 packages built:</summary>
  <ul>
    <li>ansible (python310Packages.ansible-core)</li>
    <li>ansible-later (python310Packages.ansible-later)</li>
    <li>ansible-lint (python310Packages.ansible-lint)</li>
    <li>ansible_2_12</li>
    <li>ansible_2_13</li>
    <li>kargo</li>
    <li>parsedmarc (python310Packages.parsedmarc)</li>
    <li>python310Packages.ansible</li>
    <li>python310Packages.ansible-compat</li>
    <li>python310Packages.ansible-kernel</li>
    <li>python310Packages.ansible-runner</li>
    <li>python310Packages.geoip2</li>
    <li>python310Packages.mocket</li>
    <li>python310Packages.sopel</li>
    <li>ttp (python310Packages.ttp)</li>
    <li>python311Packages.ansible</li>
    <li>python311Packages.ansible-compat</li>
    <li>python311Packages.ansible-core</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-later</li>
    <li>python311Packages.ansible-lint</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.geoip2</li>
    <li>python311Packages.mocket</li>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.ttp</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Overlap with https://github.com/NixOS/nixpkgs/pull/216511

File: pkgs/development/python-modules/mocket/default.nix [hunk 1]
```diff
@@ -1,62 +1,72 @@
...
-  # skip http tests
+  # Skip http tests
   SKIP_TRUE_HTTP = true;
-  pytestFlagsArray = [
+
+  disabledTestPaths = [
     # Requires a live Redis instance
-    "--ignore=tests/main/test_redis.py"
+    "tests/main/test_redis.py"
   ] ++ lib.optionals (pythonOlder "3.8") [
```
Comment (dotlambda) [inline pkgs/development/python-modules/mocket/default.nix:67]: Can be removed. We don't support such old versions anymore.

**Review by dotlambda — COMMENTED** [review]
Can you please squash the commits into one per package?

File: pkgs/development/python-modules/mocket/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/mocket/default.nix]: Thanks, removed.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 217295` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.pytest-ansible</li>
    <li>python311Packages.pytest-ansible</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python311Packages.sopel</li>
  </ul>
</details>
<details>
  <summary>26 packages built:</summary>
  <ul>
    <li>ansible (python310Packages.ansible-core)</li>
    <li>ansible-later (python310Packages.ansible-later)</li>
    <li>ansible-lint (python310Packages.ansible-lint)</li>
    <li>ansible_2_12</li>
    <li>ansible_2_13</li>
    <li>kargo</li>
    <li>parsedmarc (python310Packages.parsedmarc)</li>
    <li>python310Packages.ansible</li>
    <li>python310Packages.ansible-compat</li>
    <li>python310Packages.ansible-kernel</li>
    <li>python310Packages.ansible-runner</li>
    <li>python310Packages.geoip2</li>
    <li>python310Packages.mocket</li>
    <li>python310Packages.sopel</li>
    <li>ttp (python310Packages.ttp)</li>
    <li>python311Packages.ansible</li>
    <li>python311Packages.ansible-compat</li>
    <li>python311Packages.ansible-core</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-later</li>
    <li>python311Packages.ansible-lint</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.geoip2</li>
    <li>python311Packages.mocket</li>
    <li>python311Packages.parsedmarc</li>
    <li>python311Packages.ttp</li>
  </ul>
</details>

**Pull request merged**